test/eslint-test.js

```diff
@@ -161,7 +161,7 @@ describe('broccoli-lint-eslint', function() {
       yield output.build();
 
       let result = output.read();
-      expect(Object.keys(result)).to.deep.equal(['.eslintrc.lint-test.js', 'a.lint-test.js', 'b.lint-test.js']);
+      expect(Object.keys(result)).to.deep.equal(['.eslintrc.js', 'a.lint-test.js', 'b.lint-test.js']);
```

Comment:
I do find this curious... Why would these dep updates have broken this? 🤔

Reply:
I thought it was because .eslintrc.js wasn't being linted any more.

Reply:
but you're updating broccoli dependencies. those updates shouldn't influence what is being linted or not 🤔

Reply:
So, I think @rwjblue is right, it's due to the change in ESLint version. The new version is in the semver range, so existing users are getting it when they install the add-on already.

Reply:
right, seems fine, but there is (AFAICT) no reason the lock file should have updated eslint versions here

I'd suggest running `git checkout master yarn.lock && yarn`, which should reset the `yarn.lock` to whatever master has and _only_ update the deps that you explicitly changed in the package.json.

In addition, we should double check to confirm that the `.travis.yml` is using the lockfile (vs using `--no-lockfile`).

Reply:
thanks!

Reply:
done!